codectrl-server/src/lib.rs

```diff
@@ -500,6 +507,49 @@ impl Authentication for Service {
     ) -> Result<Response<Token>, Status> {
         todo!()
     }
+
+    async fn github_login(
+        &self,
+        _request: Request<GhLogin>,
+    ) -> Result<Response<LoginUrl>, Status> {
+        Ok(Response::new(LoginUrl {
```

Comment:
I guess we're using them here but we seem to be not using the `_request`, and `LoginUrl` could be a `String` instead?